recordedit/recordEdit.app.js

```diff
@@ -32,24 +53,21 @@
         'chaise.recordcreate'
     ])
 
-    .config(['$cookiesProvider', function($cookiesProvider) {
+    .config(['$compileProvider', '$cookiesProvider', '$logProvider', '$uibTooltipProvider', 'ConfigUtilsProvider', function($compileProvider, $cookiesProvider, $logProvider, $uibTooltipProvider, ConfigUtilsProvider) {
+        // angular configurations
+        // allows unsafe prefixes to be downloaded
+        // full regex: "/^\s*(https?|ftp|mailto|tel|file|blob):/"
+        $compileProvider.aHrefSanitizationWhitelist(/^\s*(https?|blob):/);
         $cookiesProvider.defaults.path = '/';
-    }])
-
-    // Configure all tooltips to be attached to the body by default. To attach a
-    // tooltip on the element instead, set the `tooltip-append-to-body` attribute
-    // to `false` on the element.
-    .config(['$uibTooltipProvider', function($uibTooltipProvider) {
-        $uibTooltipProvider.options({appendToBody: true});
-    }])
-
-    //  Enable log system, if in debug mode
-    .config(['$logProvider', function($logProvider) {
+        //  Enable log system, if in debug mode
         $logProvider.debugEnabled(chaiseConfig && chaiseConfig.debug === true);
-    }])
+        // Configure all tooltips to be attached to the body by default. To attach a
+        // tooltip on the element instead, set the `tooltip-append-to-body` attribute
+        // to `false` on the element.
+        $uibTooltipProvider.options({appendToBody: true});
 
-    .config(['ConfigUtilsProvider', function(ConfigUtilsProvider) {
-      ConfigUtilsProvider.$get().setConfigJSON();
+        // chaise configurations
+        ConfigUtilsProvider.$get().setConfigJSON();
```

Comment:
This is not really important now, but I think we want to move this to the config app that you have added. You can leave it here for now, but in the changes for getting the config from annotation, this should be moved.

Reply:
I agree with you but I didn't want to spend more time on this issue for something else that makes more sense to do with the catalog config annotation instead.